pkg/api/admin.go

```diff
@@ -21,6 +22,14 @@ func AdminGetSettings(c *m.ReqContext) {
 			if strings.Contains(keyName, "secret") || strings.Contains(keyName, "password") || (strings.Contains(keyName, "provider_config")) {
 				value = "************"
 			}
+			if strings.Contains(keyName, "url") {
```

Comment:
Since the password is part of `setting`. An even simpler solution would be to replace any occurrence of the password string in URL with `****`

Reply:
Hello @bergquist,

i'm not sure if I understand you correctly, but URL setting is read from config.ini where it is stored as plain text string. After that it is parsed to key/value store with key name being "url" and value being complete URL containing username:password. 

I'm not sure what setting you have in mind. Can you please explain it in more details?

Regards Mitja

Reply:
Sorry. My mistake. I thought setting had the database password.